hooks/playbooks/cinder_lvm_label.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Replace soft link files with source files.
+  gather_facts: false
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+
+  tasks:
+    - name: Apply the label on the resource
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command:
+        cmd: >-
+          oc label node
+          {{ cifmw_cinder_lvm_node }}
+          openstack.org/cinder-lvm=''
```

Comment:
maybe we should consider a more generic approach here? Maybe other nodes would benefit of labels in the future?

Reply:
I was going back and forth with `crd.yml` or a new one. Since most of the hooks are specific in nature opted to use CLI as it mainted idempotency . 

Let me shift to a collection and make it generic in nature.